src/data/sql-quiz.ts

```diff
@@ -1,4 +1,13 @@
 const sqlQuiz = [
+  {
+    Question: "In SQL, which of the following is not a commonly used SQL constraints",  
+    Answer: "UNIQUE",
```

Comment:
But this is a constraint. 

```suggestion
    Answer: "UPDATE",
```